Resources/Prototypes/Floof/Entities/Objects/Specific/Medical/surgery.yml

```
@@ -0,0 +1,34 @@
+- type: entity
+  name: ERT medical multitool
+  id: ERTOmnimedTool
+  parent: BaseToolSurgery
+  suffix: Admeme
```

```suggestion
  suffix: debug, DO NOT MAP
```
This ^ and I think it'd be better to rename it to something like "bluespace omnitool" or something along those lines.

---

Fair! I used that suffix because that's what the admin-only experimental RCD uses, but this also makes sense. I'll make those changes in a minute.